ansible/bootstrap.yml

```diff
@@ -8,25 +8,31 @@
       - nomad
     token: master
   tasks:
+    - name: Check if .env_override is provided
+      debug:
+        msg: "We are running in test mode and .env_override is set"
+      failed_when: not lookup( 'env', 'local_test') and (not lookup( 'env', 'env_override') and not lookup( 'env', 'TF_VAR_env_override'))
```

Comment:
Stupid question: Won't we achieve the same result from the below check? Wondering if we could reduce from 3 booleans to 2
```
failed_when: not lookup( 'env', 'local_test') and (not lookup( 'env', 'TF_VAR_env_override'))
```

Reply:
yes. Thanks!

Reply:
BTW: you can use the "suggest change"-functionality (+-) in the markdown editor to suggest directly